Documentation/states.md

```diff
@@ -1,6 +1,6 @@
 ## Unit states
 
-There are currently three cluster-level states for a unit:
+`fleetctl` uses declarative model which means it sets desired state to units. For example when you wish to start a unit `fleetctl` sets desired state inside etcd storage and `fleet` daemons connected to this storage just perform desired action.  There are currently three cluster-level states for a unit:
```

Comment:
How about:

> fleet uses a _declarative model_ to evaluate unit state. This means that operations to change the state of units (e.g. `fleetctl` commands, or calls to the fleet API) change the desired state, rather than directly performing any state change.